### ISSUE TITLE
pages index

### ISSUE THREAD

**Issue by Lemon5269** [easyops-cn/docusaurus-search-local#399]
I have encountered another issue. When creating indexes for the content under the 'pages' directory, it builds into a long string without corresponding headings. Has anyone encountered the same problem?

**Comment by tuckergordon** [comment]
Yes I'm experiencing this too - it results in search results not being able to scroll the page down to the matching string

**Comment by weareoutman** [comment]
That's because 'pages' have very flexible html output, not like docs or blog which have a predictable pattern of html output.

So currently, the content of a page is treated as a single string.

**Comment by weareoutman** [comment]
Maybe we can support opt-in to treat pages as normal docs
